main/ui/src/main/resources/fxml/vault_list.fxml

```diff
@@ -19,7 +19,7 @@
 			<contextMenu>
 				<ContextMenu>
 					<items>
-						<MenuItem text="%main.vaultlist.contextMenu.remove" onAction="#didClickRemoveVault" disable="${controller.noVaultSelected}"/>
+						<MenuItem text="%main.vaultlist.contextMenu.remove" onAction="#didClickRemoveVault" disable="${!controller.selectedVaultRemovable}"/>
```

Comment:
Can we have `disable="${controller.selectedVault.unlocked}"` instead? Avoid the listener stuff and use javafx features instead?

Reply:
Or maybe it should not just be disabled when `unlocked` but also when `processing`, then use a Binding instead of a listener. (Something like `EasyBind.select(vault).selectObject(Vault::stateProperty).map(this::allowsRemoval)`...)

Reply:
My mistake, this code is outdated...

`this.selectedVaultRemovable = Bindings.createBooleanBinding(() -> selectedVaultIsInState(LOCKED, MISSING, ERROR, NEEDS_MIGRATION), selectedVaultState);` looks good.